cmd/api/routers/routers.go

```diff
@@ -89,7 +89,7 @@ func (c *Controller) GetResourceLogs(context *gin.Context) {
 		apiVersion = fmt.Sprintf("%s/%s", group, version)
 	}
 
-	logURLs, err := c.Database.QueryLogURLs(
+	logURL, err := c.Database.QueryLogURL(
```

Comment:
Where is the container name here?  Or I assume you are going to default to one of the containers?

But don't we have to allow a user to specify which container they want?

Reply:
Yes, we need to allow the user to specify which container they want but I think we also should work when no container is specified and default to return one of them (replicating the `kubectl logs` command behavior). 

For introducing the container filtering there is a separate issue - https://github.com/kubearchive/kubearchive/issues/631